v4v/debian/dkms

```diff
@@ -1,7 +1,7 @@
 PACKAGE_NAME="v4v"
 PACKAGE_VERSION="1.0"
-MAKE[0]="make KERNEL_VERSION=$kernelver"
-CLEAN="make clean"
+MAKE[0]="'make' KERNEL_VERSION=$kernelver"
```

Comment:
I think leaving this as a as a standalone patch is probably good to draw attention to the workaround.  Would it be worth adding a comment inline for why make is in quotes?

Reply:
Yes, will do.